www/front_src/src/Resources/Filter/index.tsx

```diff
@@ -65,7 +65,46 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const Filter = (): JSX.Element => {
+type Props = Pick<
```

Comment:
@Thebarda  I think that passing the props all over really complicates the structure. Can we try using useMemo instead, or maybe follow: https://kentcdodds.com/blog/how-to-optimize-your-context-value?

Reply:
@bdauria I don't understand what you mean by "using useMemo"

Reply:
There’s a hook called useMemo that basically returns memoized computed values out of a dependency list: https://reactjs.org/docs/hooks-reference.html#usememo

Reply:
By using "useMemo", it still render the component and his own chlidren